実行してみた結果。

```
↪0 16:30 xtetsuji@pancake:~/Project/bangumi-js/apps/cli [fix-cli-202403]% npx ts-node src/app/bangumi2gcal.ts 'https://bangumi.org/tv_events/AigwQAdVkAM'
[eval].ts(17,100): error TS7006: Parameter 'incomingMessage' implicitly has an 'any' type.
[eval].ts(19,49): error TS7006: Parameter 'chunk' implicitly has an 'any' type.
[eval].ts(33,50): error TS7006: Parameter 'error' implicitly has an 'any' type.

[eval].ts(17,104): error TS7006: Parameter 'incomingMessage' implicitly has an 'any' type.
[eval].ts(19,49): error TS7006: Parameter 'chunk' implicitly has an 'any' type.
[eval].ts(33,50): error TS7006: Parameter 'error' implicitly has an 'any' type.

[eval].ts(17,165): error TS7006: Parameter 'incomingMessage' implicitly has an 'any' type.
[eval].ts(19,49): error TS7006: Parameter 'chunk' implicitly has an 'any' type.
[eval].ts(33,50): error TS7006: Parameter 'error' implicitly has an 'any' type.

[eval].ts(17,110): error TS7006: Parameter 'incomingMessage' implicitly has an 'any' type.
[eval].ts(19,49): error TS7006: Parameter 'chunk' implicitly has an 'any' type.
[eval].ts(33,50): error TS7006: Parameter 'error' implicitly has an 'any' type.

addition is not found
talent_panel is not found
related_link is not found
url.length=3058, treated=false
（Google Calendar の Event Publisher URLは長いので省略）
```

esbuild で生成した .js では上記 warning は出なかった。

```
↪0 16:41 xtetsuji@pancake:~/Project/bangumi-js/apps/cli [fix-cli-202403]% make build 
npx esbuild \
                --bundle \
                --platform=node \
                --target=esnext \
                --outfile=dist/bangumi2gcal.js \
                src/app/bangumi2gcal.ts

  dist/bangumi2gcal.js  1.7mb ⚠️

⚡ Done in 50ms
chmod +x dist/bangumi2gcal.js
wc -l dist/bangumi2gcal.js
39543 dist/bangumi2gcal.js
```

```
↪0 16:41 xtetsuji@pancake:~/Project/bangumi-js/apps/cli [fix-cli-202403]% dist/bangumi2gcal.js 'https://bangumi.org/tv_events/AigwQAdVkAM' 
addition is not found
talent_panel is not found
related_link is not found
url.length=3058, treated=false
（Google Calendar の Event Publisher URLは長いので省略）
```

--- a/apps/cli/Makefile
+++ b/apps/cli/Makefile
@@ -1,6 +1,6 @@
 # -*- makefile -*-
 
-ESBUILD = ../../node_modules/.bin/esbuild
+ESBUILD = npx esbuild
 SRC_DIR = src
 DIST_DIR = dist
 
@@ -10,12 +10,13 @@ usage:
 	@echo "Usage: make [build|tree|test]"
 
 build:
-	../../node_modules/.bin/esbuild \
+	$(ESBUILD) \
 		--bundle \
 		--platform=node \
 		--target=esnext \
 		--outfile=$(DIST_DIR)/bangumi2gcal.js \
-		$(SRC_DIR)/app/bangumi2gcal.ts 
+		$(SRC_DIR)/app/bangumi2gcal.ts
+	chmod +x $(DIST_DIR)/bangumi2gcal.js
 	wc -l $(DIST_DIR)/bangumi2gcal.js
 
 tree:

